operations/mimir-mixin/dashboards/queries.libsonnet

```
@@ -334,6 +334,17 @@ local filename = 'mimir-queries.json';
         $.panel('Index-header lazy load duration') +
         $.latencyPanel('cortex_bucket_store_indexheader_lazy_load_duration_seconds', '{%s}' % $.jobMatcher($._config.job_names.store_gateway)),
       )
+      .addPanel(
+        $.panel('Average index-header load latency') +
+        $.queryPanel(
```

Since it's a histogram, might as well use the full spectrum of percentiles
```suggestion
        $.latencyPanel('cortex_bucket_stores_gate_duration_seconds' ... )
```